Jenkinsfile

```diff
@@ -291,6 +291,8 @@ selectedArchitectures.each { suffix ->
                 '--cheribsd/build-lib32',
                 '--cheribsd/build-tests',
                 '--cheribsd/build-bench-kernels',
+                '--cheribsd/build-nocaprevoke-kernels',
+                '--cheribsd-mfs-root-kernel/no-build-nocaprevoke-kernels',
```

Comment:
Turns out this was only needed due to a cheribuild bug (it wanted to make -NOCAPREVOKE the default when enabled and that's why the -MFS-ROOT targets were unhappy; when not the default it happily automatically just doesn't build them)